CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+## Setup for Code Contributions
+
+### Prerequisites
+
+##### MacOS
+- XCode Command Line Tools `xcode-select --install`
+- If you already have the full Xcode installed, go to menu `Xcode -> Open Developer Tool -> More Developer Tools....`
+- Install Node.js >= 16
+
+##### Windows
+- Install current version of Python3
+- Install Visual Studio Code and add the Microsoft C/C++ extension
+- Install Node.js >= 16
+
+##### Linux
+
+- Install Python > 3.7
+- Install `make`, `gcc`, and `g++`
+- Install Node.js >= 16
+
+#### Repository Setup
+- Fork the project.
```

Comment:
Hmm, this won't create the beta branch. I'm not sure how to resolve that.

Reply:
Maybe if we have them run `git pull upstream beta`. I think it'll work, but I'll test it.

Reply:
Well, if we explicitly tell them NOT to ignore that and maybe imbed an image or something?